src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
```

Comment:
The color property should not be in the hover selector of the card. It should be in the hover selector of the card title to change the card title text color to `#34568b`.